### PR TITLE
word-smith

### PR DESCRIPTION
**Pull request by Rick-Anderson**
# word-smith
## Summary

edit clean up,, kill dead wood, put 3 commands on 3 lines.

Fixes 
## Suggested Reviewers

@mairaw

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C30%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609300226411876-1095%5Cworkflow_report.html).

File: samples/core/getting-started/unit-testing-using-dotnet-test/README.md [hunk 1]
```diff
@@ -2,41 +2,40 @@ Unit Testing using dotnet test Sample
...
 Key Features
 ------------
 
-This sample demonstrates creating a library and writing effective unit tests
-that validate the features in that library. 
+This sample demonstrates creating a library and writing effective unit tests that validate the features in that library. 
 
-The example provides a service that indicates whether a number is or is not
-a prime number.
+The example provides a service that indicates whether a number prime.
```
Comment (mairaw) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: a number is prime?

File: samples/core/getting-started/unit-testing-using-dotnet-test/README.md [hunk 2]
```diff
@@ -2,41 +2,40 @@ Unit Testing using dotnet test Sample
...
+`dotnet run` runs the executable. 
 
-To run the tests, change to the `tests/PrimeService.Tests` directory and
-type the following three commands:
+To run the tests, navigate to the `tests/PrimeService.Tests` directory and type the following commands:
 
-`dotnet restore`
-`dotnet build`
-`dotnet test`
+```dotnet restore
```
Comment (mairaw) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: new line before dotnet restore

**Review by mairaw — CHANGES_REQUESTED** [review]
LGTM but I found a few minor issues

File: samples/core/getting-started/unit-testing-using-dotnet-test/README.md (hunk 1 above)
Comment (Rick-Anderson) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: Isn't that what it does? True if a number is prime, else false?

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C30%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609300318557751-1095%5Cworkflow_report.html).

File: samples/core/getting-started/unit-testing-using-dotnet-test/README.md [hunk 3]
```diff
@@ -2,41 +2,41 @@ Unit Testing using dotnet test Sample
...
-`dotnet build`
-`dotnet test`
+```
+dotnet restore
+dotnet build
+dotnet test
+```
 
-`dotnet test` runs all the configure tests 
+`dotnet test` runs all the configure tests.
```
Comment (svick) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: What does "configure" mean here? Should it be just:

> `dotnet test` runs all the tests.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C30%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609301717417571-1095%5Cworkflow_report.html).

File: samples/core/getting-started/unit-testing-using-dotnet-test/README.md (hunk 3 above)
Comment (mairaw) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: @BillWagner what did you intend to say here initially? configured tests perhaps?

Comment (BillWagner) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: "configured tests" is correct.

Comment (mairaw) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: @Rick-Anderson can you also please fix this and the other feedback I gave you and then we can merge this. Thanks!

File: samples/core/getting-started/unit-testing-using-dotnet-test/README.md (hunk 1 above)
Comment (Rick-Anderson) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: depends on what your definition of is is.

File: samples/core/getting-started/unit-testing-using-dotnet-test/README.md (hunk 3 above)
Comment (Rick-Anderson) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: configured was all I fixed, did I miss something?

**Review by mairaw — APPROVED** [review]
LGTM. Thanks @Rick-Anderson!

File: samples/core/getting-started/unit-testing-using-dotnet-test/README.md (hunk 3 above)
Comment (mairaw) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: I missed your earlier fix. My bad!

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C1%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610010045183835-1095%5Cworkflow_report.html).

**Pull request merged**